src/types.ts

```diff
@@ -223,5 +226,21 @@ export interface FilterRule {
   color?: string
 }
 
-export const STORE_TYPES = ["credits", "marks"] as const
-export type StoreType = typeof STORE_TYPES[number]
+export interface FormFilterRule {
+  character: string
+  item: string
+  blessing: string
+  perk: string
+  store: string
+  minBlessingRarity: string
+  minPerkRarity: string
+  minStats: string
+  minRating: string
+  isOpen: boolean
+}
+
+export enum Size {
+  Small,
+  Medium,
+  Large
+}
```

Comment:
Not a fan of enums, this could be a string union

Reply:
Fixed, re-implemented as string union